paket.lock

```diff
@@ -123,8 +123,8 @@ NUGET
       xunit.core (2.5.3)
     xunit.abstractions (2.0.3)
     xunit.analyzers (1.4)
-    xunit.assert (2.5.3)
-      NETStandard.Library (>= 1.6.1)
+    xunit.assert (2.6.1)
```

Comment:
I don't know enough of this change to judge these version fixes, but... shouldn't this stay at 2.5.3? Or is this for the internal tests only?

Reply:
This is for internal use only, yes.